engine/audio/private/fmod_debug.cpp

```diff
@@ -4,7 +4,14 @@
 #include "fmod_errors.h"
 #include "log.hpp"
 #include <iostream>
-#include <string>
+void CHECKRESULT_fn(FMOD_RESULT result, MAYBE_UNUSED const char* file, int line)
+{
+    if (result != FMOD_OK)
+    {
+        bblog::error("FMOD ERROR: audio_module.cpp [Line {0} ] {1} - {2}", line, static_cast<int>(result), FMOD_ErrorString(result));
+        throw FMOD_ErrorString(result);
```

Comment:
Might be better to throw std::runtime_error(FMOD_ErrorString(result)). This allows us to handle exceptions like this with try catch and probably prints better debug messages